### PR TITLE
Start OIDC tutorial docs

### PR DESCRIPTION
**Pull request by fredx30**
Inlcudes add a turorial about
- Setting up azure ad as an idp for wharf.
- Setting up https://oath.tools as a test for the idp configuration.
-

### CODE REVIEW

File: docs/tutorials/oidc/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# OpenID Connect (OIDC)
```
Comment (jilleJr) [inline docs/tutorials/oidc/README.md:1]: You need to also link to this page through one of the side panels. It doesn't automatically detect new pages.

File: docs/tutorials/oidc/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+## Provider setup
+Setup of full-featured OIDC contains many feature flags. This can be 
+confusing to set up. Use the 
+following settings and words of warning to get started.
+
+### Create an azure app registration
+
+In azure create a new app registration as a 'web' application. The web 
+platform is used specifically for the oath-tools callback. OBS! It is not 
+used in wharf, wharf uses the SPA type platform.
```
Comment (jilleJr) [inline docs/tutorials/oidc/README.md:31]: nit: wharf->Wharf

```suggestion
used in Wharf, Wharf uses the SPA type platform.
```

File: docs/tutorials/oidc/README.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+Use the callback url `https://oauth.tools/callback/code` for testing. In the 
+next step of the tutorial this is used to verify the flow works.
+![azure-config-auth](azure-config-auth.png)
+
+Under the menu item `Certificates and Secrets` its possible to create a 
+client secret. This is not needed for SPA platform that 
+wharf uses, however it is needed when using the (WEB) type of platform that
```
Comment (jilleJr) [inline docs/tutorials/oidc/README.md:38]: nit: wharf->Wharf

```suggestion
Wharf uses, however it is needed when using the (WEB) type of platform that 
```

**Review by jilleJr — CHANGES_REQUESTED** [review]

File: docs/tutorials/oidc/README.md (hunk 1 above)
Comment (fredx30) [reply docs/tutorials/oidc/README.md:1]: https://github.com/iver-wharf/iver-wharf.github.io/pull/92/files#diff-e9741d749788e4bfb8b76aff7a4e39432c2a0f6ab582191fb588f0274f704faeR7

**Review by Alexamakans — APPROVED** [review]

**Pull request merged**